core/slice.go

```diff
@@ -843,6 +857,7 @@ func (sl *Slice) NewGenesisPendingHeader(domPendingHeader *types.Header) {
 	genesisTermini := []common.Hash{genesisHash, genesisHash, genesisHash, genesisHash}
 	if sl.hc.Empty() {
 		sl.phCache.Add(sl.config.GenesisHash, types.PendingHeader{Header: domPendingHeader, Termini: genesisTermini})
+		sl.phCacheSeal.Add(domPendingHeader.SealHash(), *domPendingHeader)
```

Comment:
we don't need any of these phCacheSeal anymore, so you can remove most of the PR pretty much

Reply:
Can you please clarify what changes were made to make it not needed? Seems we still use the ph sealhash to pull the right ph when verifying a result from the miner (in the case of stale shares).

Reply:
this work is not complete, since right now we use the proxy to mine using GPU, this method is not needed. You use the get pending header and use receiveminedHeader after that to submit